deploy/registry/local/service.yaml

```diff
@@ -13,6 +13,7 @@ metadata:
   labels:
     app: che-plugin-registry
   name: che-plugin-registry
+  namespace: operators
```

Comment:
Why do you force installing the plugin registry in the `operators` namespace ?

Afair this namespace doesn't exist on OpenShift.

Reply:
it's just the same namespace where DevWorkspace Controller is deployed.

Do you think it's better to deploy Plugin Registry in it's own namespace or default one?
Also, where do you think DevWorkspace Controller should be deployed? We can precreate some namespace like `devworkspace-controller` for controller and plugin registry, WDYT?

P.S. I considered an option where user specify namespace for `kubectl apply -f ... -n ` BUT seems it's not possible since we need to know namespace before executing that command  for ClusterRoleBinding

Reply:
> it's just the same namespace where DevWorkspace Controller is deployed.

Only in the raw `Kubernetes` case no ?

> Do you think it's better to deploy Plugin Registry in it's own namespace or default one?

Not necessarily

> Also, where do you think DevWorkspace Controller should be deployed? We can precreate some namespace like devworkspace-controller for controller and plugin registry, WDYT?

In fact when you deploy manually with raw yamls, you can depoy in any namespace, and this will only watch CRs in this namespace.

But I had also started setting up OperatorHub files (CSV) so that it can be installed through OperatorHub in `AllNamespaces` mode (which should be the preferred way in the future I assume)

> P.S. I considered an option where user specify namespace for kubectl apply -f ... -n BUT seems it's not possible since we need to know namespace before executing that command for ClusterRoleBinding

In such a case we could provide a script file that deploys all the file withing the namespace provided as an argument no ?

Reply:
> In such a case we could provide a script file that deploys all the file withing the namespace provided as an argument no ?

I would consider it as a further improvement.